src/xcode/ENA/ENA/Source/AppDelegate.swift

```diff
@@ -377,6 +382,11 @@ extension AppDelegate: ENATaskExecutionDelegate {
 			taskScheduler.scheduleBackgroundTask(for: .fetchTestResults)
 		}
 
+		self.taskScheduler.notificationManager.presentNotification(
+			title: "BGTask event received",
+			body: "\(ENATaskIdentifier.fetchTestResults.backgroundTaskSchedulerIdentifier)",
+			identifier: ENATaskIdentifier.fetchTestResults.rawValue)
+
```

Comment:
Not really part of this PR?

Reply:
Shouldn't have been committed, was used for debugging the BGTasks